src/commands/build/wranglerjs/mod.rs

```diff
@@ -267,7 +269,7 @@ fn install() -> Result<PathBuf, failure::Error> {
         wranglerjs_path.path()
     };
 
-    run_npm_install(wranglerjs_path.clone()).expect("could not install wranglerjs dependencies");
+    run_npm_install(&wranglerjs_path.clone()).expect("could not install wranglerjs dependencies");
```

Comment:
I don't think you need the clone() call here; you can borrow wranglerjs_path (I tested this and it worked fine!)
```suggestion
    run_npm_install(&wranglerjs_path).expect("could not install wranglerjs dependencies");
```